src/lib/utils/language.js

```diff
@@ -10,9 +10,15 @@
  */
 const languageNames = {
   en: 'English',
-  // pl: 'Polish', TODO: Uncomment when PL language is prod-ready.
+  pl: 'Polish', // TODO: Uncomment when PL language is prod-ready.
```

Comment:
Will you comment this back out before landing the PR?

Reply:
Yes, that's the plan.